doc/tutorial/compile.rst

```diff
@@ -1,7 +1,7 @@
 Build a demo project
 ********************
 
-Prequisites
+Prerequisites
 ===========
```

Comment:
I'm afraid rst requires that the underlining must be at least as long as the text...

```suggestion
=============
```

Reply:
You never stop learning ;)